From f9497cbb465453d5c8f895f7b70c19f7b0d0d23c Mon Sep 17 00:00:00 2001
From: Justin Kim <justin.kim@collabora.com>
Date: Tue, 2 Jan 2018 20:17:46 +0900
Subject: [PATCH] Fix #include path for wintime.h

`wintime.h` is installed `${includedir}/srt/win`
so `platform_sys.h` shouldn't use an absolute path
for this header.

Signed-off-by: Justin Kim <justin.kim@collabora.com>
---
 common/win/wintime.h   | 3 +--
 scripts/haisrt.pc.in   | 2 +-
 srtcore/platform_sys.h | 2 +-
 3 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/common/win/wintime.h b/common/win/wintime.h
index af810b6..cc37b52 100644
--- a/common/win/wintime.h
+++ b/common/win/wintime.h
@@ -4,7 +4,6 @@
 #include <winsock2.h>
 #include <windows.h>
 #include <time.h>
-#include "haicrypt.h"
 
 #ifdef __cplusplus
 extern "C" {
@@ -34,7 +33,7 @@ struct timezone
 
 void timeradd(struct timeval *a, struct timeval *b, struct timeval *result);
 
-HAICRYPT_API int gettimeofday(struct timeval* tp, struct timezone* tz);
+int gettimeofday(struct timeval* tp, struct timezone* tz);
 
 #ifdef __cplusplus
 }
diff --git a/scripts/haisrt.pc.in b/scripts/haisrt.pc.in
index beb103b..9009f9b 100644
--- a/scripts/haisrt.pc.in
+++ b/scripts/haisrt.pc.in
@@ -8,5 +8,5 @@ Description: SRT library set
 Version: @SRT_VERSION@
 Libs: -L${libdir} -l@TARGET_srt@ @IFNEEDED_LINK_HAICRYPT@ @IFNEEDED_SRTBASE@ @IFNEEDED_SRT_LDFLAGS@
 Libs.private: @SRT_LIBS_PRIVATE@
-Cflags: -I${includedir}
+Cflags: -I${includedir} -I${includedir}/srt
 Requires.private: @SSL_REQUIRED_MODULES@
diff --git a/srtcore/platform_sys.h b/srtcore/platform_sys.h
index 5ffa374..f2cb09b 100644
--- a/srtcore/platform_sys.h
+++ b/srtcore/platform_sys.h
@@ -8,7 +8,7 @@
    #include <windows.h>
    #include <inttypes.h>
    #include <stdint.h>
-   #include <win/wintime.h>
+   #include "win/wintime.h"
    #if defined(_MSC_VER)
       #pragma warning(disable:4251)
    #endif
-- 
2.11.0

